l10n/en-US/viewer.properties

```diff
@@ -263,3 +263,13 @@ editor_stamp_add_image.title=Add image
 editor_free_text2_aria_label=Text Editor
 editor_ink2_aria_label=Draw Editor
 editor_ink_canvas_aria_label=User-created image
+
+# Alt-text dialog
+editor_alt_text_dialog_label=Choose an option
+editor_alt_text_dialog_description=Alternative text (alt text) helps when people can’t see the image or when it doesn’t load.
+editor_alt_text_add_description_label=Add description
```

Comment:
I see "Enter a description" in the copy doc.

Reply:
Yep the I saw the same but we've two times "Add description". I just asked to Dasha on slack what's the correct one.